### DISCUSSION TITLE
Tailwind class is not working in vite+react as everything is done as per instruction

### DISCUSSION THREAD

**Discussion by sujal58** [tailwindlabs/tailwindcss#13553] [Help]
![Screenshot (33)](https://github.com/tailwindlabs/tailwindcss/assets/95696038/012f0efb-dceb-44ff-bd9c-dffe1254c0db)

**Comment by wongjn** [comment]
It doesn't seem like you've followed [the instructions for Vite](https://tailwindcss.com/docs/guides/vite):
- You seem to have tried to configure PostCSS in `vite.config.js` but if you followed step 2, you would have had a `postcss.config.js` created for you.
- Consider checking you have followed step 4. You should have the `@tailwind` directives in a CSS file that is `import`ed in you project code.

**Reply by rohan-v8rma** [reply]
> * Consider checking you have followed step 4. You should have the `@tailwind` directives in a CSS file that is `import`ed in you project code.

This was the problem for me, I had done everything, except for uncommenting the import of `index.css` that was generated when I scaffolded the Vite project, where I wrote the `tailwind` directives into my project.

**Comment by dhruvjain23** [comment]
Make sure that `./src/index.css` file must contains 

`@tailwind base;
@tailwind components;
@tailwind utilities;
`
